silverback/_cli.py

```diff
@@ -36,11 +42,7 @@ def _account_callback(ctx, param, val):
 
 
 def _network_callback(ctx, param, val):
-    if val:
-        os.environ["SILVERBACK_NETWORK_CHOICE"] = val
-    else:
-        val = os.environ.get("SILVERBACK_NETWORK_CHOICE", "")
-
+    os.environ["SILVERBACK_NETWORK_CHOICE"] = val.network_choice
```

Comment:
question: So is the callback only called when the network option is explicitly used?